### PR TITLE
add the manual of strategy in chart

### PR DESCRIPTION
**Pull request by my-git9**
add the manual of strategy in chart
 
/kind cleanup

### CODE REVIEW

**Comment by RainbowMango** [comment]
@Poor12 Please help to review it.

File: charts/README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ $ helm install karmada-scheduler-estimator -n karmada-system ./charts
...
 | `etcd.internal.resources`                 | Resource quota of the etcd                                                                                                                                                                                                                    | `{}`                                                                                                                                                       |
+| `etcd.strategy`                           | Strategy of the etcd                                                                                                                                                                                                                          | `{"type": "RollingUpdate", "rollingUpdate": {"maxUnavailable": "0", "maxSurge": "50%"} }`                                                                  |
```
Comment (Poor12) [inline charts/README.md:160]: etcd doesn't appear to have a rollingupdate policy configured. I guess the default value is not {"maxUnavailable": "0", "maxSurge": "50%"}. BTW, others should be checked that doc and yaml files keep the same.

**Pull request merged**